content/home/examples/an-application.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: An Application
+title: Bir Uygulama
 order: 2
 domid: todos-example
 ---
 
-Using `props` and `state`, we can put together a small Todo application. This example uses `state` to track the current list of items as well as the text that the user has entered. Although event handlers appear to be rendered inline, they will be collected and implemented using event delegation.
+`props` ve `state` kullanarak, küçük bir Todo uygulaması oluşturabiliriz. Bu örnekte `state`, hem maddelerin bulunduğu güncel listeyi hem de kullanıcının girmiş olduğu metni takip etmek için kullanılıyor. Olay yöneticileri, satıriçi oluşturuluyor gibi gözükmelerine rağmen, olay delagasyonu ile toplanıp gerçekleştirilir.
```

Comment:
`current list of items` -> `öğelerin bulunduğu güncel liste`.  Which one is more proper for the translation of `item`?  Madde or oge or eleman ? What do you think ?

Reply:
or even more clear,`öğelerin güncel listesi`

Reply:
I liked `öğe` much better. But I think we can keep `bulunduğu`. It's not making it much longer but a little clearer I think.